web/src/main/ui/app/components/flows-new/edit-flow/mapping/ui/mapping-ui.component.html

```diff
@@ -29,93 +27,93 @@
-
-      <!-- Row for each entity property -->
-      <div *ngFor="let prop of targetEntity?.definition.properties; let i = index;">
-
-        <div class="entity-prop-container {{ 'entity-prop-container-' + prop.name }} ">
-          <span class="prop-name">{{prop.name}}</span>
-          <span class="prop-type">{{prop.datatype}}</span>
-          <span class="entity-icons">
-            <span class="entity-icon" tooltip="Primary Key" container="body"><i class="fa fa-key fa-fw" [ngClass]="isPrimaryKey(prop.name) ? 'entity-icon-selected' : 'icon-hide'"></i></span>
-            <span class="entity-icon" tooltip="Element Range Index" container="body"><i class="fa fa-bolt fa-fw" [ngClass]="hasElementRangeIndex(prop.name) ? 'entity-icon-selected' : 'icon-hide'"></i></span>
-            <span class="entity-icon" tooltip="Path Range Index" container="body"><i class="fa fa-code fa-fw" [ngClass]="hasRangeIndex(prop.name) ? 'entity-icon-selected' : 'icon-hide'"></i></span>
-            <span class="entity-icon" tooltip="Word Lexicon" container="body"><i class="fa fa-krw fa-fw" [ngClass]="hasWordLexicon(prop.name) ? 'entity-icon-selected' : 'icon-hide'"></i></span>
-            <span class="entity-icon" tooltip="Required Field" container="body"><i class="fa fa-exclamation fa-fw" [ngClass]="isRequired(prop.name)? 'entity-icon-selected' : 'icon-hide'"></i></span>
-            <span class="entity-icon" tooltip="Personally Identifiable Information" container="body"><i class="fa fa-lock fa-fw" [ngClass]="isPII(prop.name)? 'entity-icon-selected' : 'icon-hide'"></i></span>
-          </span>
-        </div>
-
-      </div><!-- ngFor -->
-    </div><!-- target -->
-
-  </div><!-- map-container -->
-
-</div><!-- layout-padding -->
+  <mat-table id="source-new-table" 
+             [dataSource]="dataSource"
+             #table class="source-props" 
+             matSort>
+    <ng-container matColumnDef="key">
+      <mat-header-cell *matHeaderCellDef mat-sort-header>Name</mat-header-cell>
+      <mat-cell *matCellDef="let srcProp">{{'@'+srcProp.key}}</mat-cell>
+    </ng-container>
+    <ng-container matColumnDef="val">
+      <mat-header-cell *matHeaderCellDef mat-sort-header>Value</mat-header-cell>
+      <mat-cell *matCellDef="let srcProp">{{ isQuoted(srcProp.type) ? '"' : '' }}{{ srcProp.val | slice:0:valMaxLen }}{{ srcProp.val.length > valMaxLen ? '...' : '' }}{{ isQuoted(srcProp.type) ? '"' : '' }}</mat-cell>
+    </ng-container>
+    <mat-header-row *matHeaderRowDef="displayedColumns; sticky:true"></mat-header-row>
+    <mat-row *matRowDef="let row; columns: displayedColumns;"></mat-row>
+  </mat-table>      
+</div>
+
+<mat-divider [vertical]="true"></mat-divider>
+
+
+
+  <div class="Entities_Tbl">
+      <div class="item-title"> 
+    <div class="item-type">Entity</div>
+    <span class="help-icon">
+        <a href="https://marklogic.github.io/marklogic-data-hub/refs/index-settings/" target="_blank"><i class="fa fa-question-circle fa-lg"></i></a>
```

Comment:
I didn't see anything in 3010 about having help icons here. I think it's a good idea, but I don't think this is the right page to go to for "Entity", as those are DHF4 docs. I'd say to point to the mapping page in the DHF5 docs, but that page is out-of-date with this. So for now, I wouldn't put any links in.